src/loaders/prowizard/tp1.c

```diff
@@ -193,7 +193,7 @@ static int test_tp1(const uint8 *data, char *t, int s)
 	}
 
 	/* pattern list size */
-	len = data[281];
+	len = readmem16b(data + 280) + 1;
 	if (len == 0 || len > 128) {
```

Comment:
The `len == 0` check here is stale now, isn't it?

Reply:
Not only is it stale but, since 0 seems to mean 1 position in the order list, it probably was never correct.